docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md

```diff
@@ -159,3 +159,47 @@ You can create a connection to a node as follows:
 ## Connect to the universe
 
 For information on how to connect to the universe from the Kubernetes cluster, as well as remotely, see [Connect YugabyteDB clusters](../../../deploy/kubernetes/clients/#connect-tls-secured-yugabytedb-cluster-deployed-by-helm-charts).
+
+### Create common TServer service for zones
+
+By default each zone gets its own TServer service which can be used to connect to the universe. Optionally, you can create a common service across these zones as follows. 
+
+Note that this requires all the zone deployments to be in the same namespace.
+
+1. Set the following Kubernetes overrides to add universe name label on the TServer pods. You can do this [during universe creation](#configure-helm-overrides) or by [modifying the Kubernetes overrides](../../manage-deployments/edit-helm-overrides/) of an existing universe.
+   ```yaml
+   tserver:
+     podLabels:
+       universe-name: yb-k8s
+   ```
+1. Save following block as `yb-tserver-common-service.yaml` file. You can modify the service type, annotations, and the label selector according to your needs.
```

Comment:
Where does the file go? Is it somehow assumed?

Reply:
We use this temporary file to save the block, make any modifications, and then apply it in the next step. 
After that the we don't care what happens with the file basically.